src/tv2-common/inewsConversion/converters/ParseCue.ts

```diff
@@ -926,6 +957,32 @@ function findGraphicDesignConfiguration(
 	)
 }
 
+export function createCueDefinitionGraphicSchema(
+	schema: string,
+	config: TV2ShowStyleConfig
+): CueDefinitionGraphicSchema | undefined {
+	const schemaConfiguration = findGraphicSchemaConfiguration(config, schema)
+	if (!schemaConfiguration) {
+		return undefined
+	}
+
+	return literal<CueDefinitionGraphicSchema>({
+		type: CueType.GraphicSchema,
+		schema: schemaConfiguration.VizTemplate,
+		iNewsCommand: '',
+		start: {
+			frames: 1
```

Comment:
What is the reason for starting at frame 1 instead of 0?

Reply:
Jacobs wants it to start on the first frame, so I put it as `frames: 1`. 
Do you know if there is a difference between 1 and 0, because if not, I would also prefer the 0

Reply:
There's a difference of 40ms 🙂 but I don't know of any reason why we would have to specifically start from frame 1 or later. When I hear _the first frame_, that's `frames: 0` to me.

Reply:
That's a good point. Will update